python/lsst/daf/butler/core/repoTransfers.py

```diff
@@ -57,10 +57,11 @@ class RepoTransferFormatConfig(ConfigSubset):
 class FileDataset:
     """A struct that represents a dataset exported to a file.
     """
-    __slots__ = ("ref", "path", "formatter")
+    __slots__ = ("refs", "path", "formatter")
 
-    ref: DatasetRef
-    """Registry information about the dataset (`DatasetRef`).
+    refs: List[DatasetRef]
+    """Registry information about the dataset. (`DatasetRef` or `list` of
+    `DatasetRef`).
```

Comment:
This is the doc for the attribute, not the ctor arg, so I think the type is just "`list` of `DatasetRef`" here.